website/docs/guides/guides-upgrade.md

```diff
@@ -45,77 +45,124 @@ If you executed the previous step already on your server, then obviously nothing
 
 It is recommended to lock your installation and avoid changes during/after the backup.
 This can either be done by either of the following:
+
 * Lock LanSuite by enabling the corresponding option under "Admin-Page" -> "Common Settings" -> "Lock LanSuite page"
-* Shut down your web server
+* Shut down public access to your web server
 * Move index.php to a different location
 
-### Back up the database
-
-Also multiple variants here:
-* LS XML export ("Admin-Page" -> "Export" -> "XML - Full Database")
-* MySQL commandline dump (Something like `mysqldump --quick -u<ls user> -p <dbname> > exportfile.sql`)
-* PHPMyAdmin - if installed
-* Any tool provided by your hoster
-
-Backing up your database using `mysqldump` is recommended, as both, LanSuite and PHPMyAdmin, may hit server limits on larger installations.
-
-### Backup the current installation
-
-* If you have command line access: `tar cfvz ls_backup.tar.gz <ls_folder>` or `cp -R <ls_folder> <ls_folder>.backup`
-* Download the whole folder with (S)FTP or any other remote access tool you have available
-
 ## Upgrade
 
 ### Copy over files from an old installation
 
-Things of relevance here:
+Replace the new files with the old ones.
+
+A few files and locations are unique to your installation.
+Those should be kept.
+Like:
+
 * The base configuration file from `inc/base/config.php`
 * Anything under `ext_inc`
 * Custom designs from `designs/`
 
 Copy them to the same place on the new installation, overwriting everything there.
 
-### Check file access rights
-
-Depending on the user you did the installation with it may be required to reset the file and folder access rights.
-
 ### Run specific release tasks
 
 It may be required to run additional steps to prepare everything for the new version.
-These will be detailed in either `README.md` or `UPGRADE.md`
+Please check the upgrade guide to your specific version below.
+
+### Execute LanSuite upgrade logic
+
+Visit "Admin-Page" -> "Lansuite updaten / reparieren" ->> "Datenbank updaten und verwalten".
+This is also available at `http(s)://<your-domain>/index.php?mod=install&action=db`.
```

Comment:
We may need to add also something for menu updates. I noticed that changes to the `menu.xml` files are apparently not taken over automatically

Reply:
How do you do it? Where? Would you mind adding this step?

Reply:
I'm still figuring that out myself.
Re-Running the install wizzard as marked above could be the solution.
Will get back to you if I find something, would suggest to add a TODO and not wait for us to figure that out

Reply:
I created https://github.com/lansuite/lansuite/issues/893 and won't add a TODO